components/infieldbutton/stories/infieldbutton.stories.js

```diff
@@ -1,80 +1,86 @@
 import { default as IconStories } from "@spectrum-css/icon/stories/icon.stories.js";
+import { uniqueUiIconBaseNames } from "@spectrum-css/icon/stories/utilities.js";
+import { Sizes, withDownStateDimensionCapture } from "@spectrum-css/preview/decorators";
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isActive, isDisabled, isFocused, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
+import { isActive, isDisabled, isHovered, isQuiet, size } from "@spectrum-css/preview/types";
 import metadata from "../dist/metadata.json";
 import packageJson from "../package.json";
 import { InfieldButtonGroup } from "./infieldbutton.test.js";
-import { Template } from "./template.js";
+import { InfieldButtonGroupVariant, InfieldButtonIcons, Template } from "./template.js";
 
 /**
- * The in-field button component is a button used inside a text field.
+ * In-field buttons are used to represent actions within input fields. They’re currently used inside the [combo box,](/docs/components-combobox--docs) [number field,](/docs/components-stepper--docs) and [search field.](/docs/components-search--docs)
  */
 export default {
 	title: "In-field button",
 	component: "InFieldButton",
 	argTypes: {
 		size: size(["s", "m", "l", "xl"]),
 		isQuiet,
-		position: {
-			name: "Position",
-			type: { name: "string", required: true },
-			table: {
-				type: { summary: "string" },
-				category: "Component",
-			},
-			options: ["left", "right", "top", "bottom"],
-			control: "select"
-		},
 		iconName: {
-			...IconStories?.argTypes?.iconName ?? {},
-			if: false,
+			...IconStories?.argTypes?.uiIconName ?? {},
+			options: uniqueUiIconBaseNames,
```

Comment:
I may have missed it in the thread, but I was under the impression that design only has use cases for the 4 supported icons. I think if that's the case, we should remove this `uniqueUiIconBaseNames` logic, and just do: 

```
options: ["ChevronDown", "Cross", "Dash", "Add"]
```

This change would then mean also updating the description to remove the arrow and asterisk stuff.